### PR TITLE
Add golang wrapper for indy-vdr.

### PR DESCRIPTION
**Pull request by pfeairheller**
Exposed Client with public methods for all features exposed as endpoint in the proxy.  Also added write methods
for signing messages to write to the ledger.

Signed-off-by: pfeairheller <pfeairheller@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
@andrewwhitehead - can you please take a look at this.  @troyronda -- perhaps you could as well?

File: wrappers/golang/cmd/demo/demo.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/pfeairheller/indy-vdr/wrappers/golang/crypto"
```
Comment (troyronda) [inline wrappers/golang/cmd/demo/demo.go:10]: These imports should be updated to this hyperledger repo.

File: wrappers/golang/go.mod [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module github.com/pfeairheller/indy-vdr/wrappers/golang
```
Comment (troyronda) [inline wrappers/golang/go.mod:1]: Should be updated to this hyperledger repo.

File: wrappers/golang/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white&style=for-the-badge)](https://pkg.go.dev/github.com/pfeairheller/indy-vdr/wrappers/golang/vdr)
```
Comment (troyronda) [inline wrappers/golang/README.md:1]: Should be updated to this hyperledger repo.

File: wrappers/golang/go.mod (hunk 2 above)
Comment (pfeairheller) [reply wrappers/golang/go.mod:1]: Good catch, thank you!  Pushing a fix shortly

**Comment by andrewwhitehead** [comment]
Looks good, thanks!

**Pull request merged**